lib/trace.c

```diff
@@ -70,13 +69,15 @@ static const char     trace_magic_symbol_name[] = "trace_magic_symbol";
 
 /* single buffer for now */
 
+enum {BOOTLOG_BUF_SIZE = (1 << 16)}; /* 64K */
```

Comment:
\*nitpicking\*  Spaces after `{` and before `}`. See M0_LOCALITY_LOCKERS_NR.

\*suggestion\* `/* The bootlog buffer size should be aligned with the page size. See the preconditions in m0_trace_init. */` I'm not sure though if it is really necessary.

Reply:
Agree, added. Thanks!